src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -309,7 +309,7 @@ The following example implements the same using `navigate()` in a React `<Form /
 import React from 'react';
 import { navigate } from 'astro:transitions/client';
 
-export default function ClickToNavigate({ to }) {
+export default function Form() {
```

Comment:
Missing `to` prop ⬇️ 
```suggestion
export default function Form({ to }) {
```

Reply:
Hi! Thank you for your reply!
Why would we include a "to" prop and not use it? 🤔

Reply:
your right, `to` can be removed 😄 , suggestion cancelled